Bottom: 8764d7dc86ee7507babfe8b386b066b22c61c945
Top:    60c313e57909d1dfac2bf78c08456c848e6b4346
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 19:13:36 +0800

Refresh of autofs-5.0.7-fix-compile-error-with-heimdal-support-enabled.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 3228d6b..17241c0 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -57,6 +57,7 @@
 - fix a couple of compiler warnings.
 - add after sssd dependency to unit file.
 - dont start readmap unless ready.
+- fix compile error with heimdal support enabled.
 
 25/07/2012 autofs-5.0.7
 =======================
